src/lib.rs

```diff
@@ -26,12 +26,13 @@
 //!
 //! # Dependencies
 //!
-//! This crate by default has no dependencies and is `#![no_std]` compatible.
-//! The following Cargo features, however, can be used to enable various pieces
-//! of functionality.
+//! By default, this crate depends on nothing but `libstd` and cannot generate
+//! any `Uuid`s. You need to enable the following Cargo features to enable
+//! various pieces of functionality:
 //!
 //! * `std` - adds in functionality available when linking to the standard
```

Comment:
We can probably just remove `std` from this list, since we're implying that you need to enable these features to get functionality,